lib/Zonemaster/CLI.pm

```diff
@@ -618,6 +618,10 @@ sub run {
         if ( $self->test and @{ $self->test } > 0 ) {
             foreach my $t ( @{ $self->test } ) {
                 my ( $module, $method ) = split( '/', $t, 2 );
+                if ( $module =~ /^([A-Za-z_]+)[0-9]+$/ ) {
+                    $method = $module;
+                    $module = $1;
+                }
                 if ( $method ) {
                     Zonemaster::Engine->test_method( $module, $method, Zonemaster::Engine->zone( $domain ) );
                 }
```

Comment:
I think it would be clearer if $method is $testcase.

Reply:
I thought about that too. There is a logic to use `method` since this is the name of the implemented method in the code. I don't have a strong opinion on which word to use.

Reply:
There are many methods. $testcase makes it easier to read the code.

Reply:
Todays code permits `basic/basic01` and `BASIC/basic01`, but it does not permit `basic/BASIC01`. I see that as a bug. Could that be fixed in this PR?

Reply:
Maybe this could be done in an independent PR, see #343.